examples/Billing/GetInvoices.php

```diff
@@ -0,0 +1,212 @@
+
+    public static function main()
+    {
+        // Either pass the required parameters for this example on the command line, or insert them
+        // into the constants above.
+        $options = (new ArgumentParser())->parseCommandArguments([
+            ArgumentNames::CUSTOMER_ID => GetOpt::REQUIRED_ARGUMENT,
+            ArgumentNames::BILLING_SETUP_ID => GetOpt::REQUIRED_ARGUMENT
+        ]);
+
+        // Generate a refreshable OAuth2 credential for authentication.
+        $oAuth2Credential = (new OAuth2TokenBuilder())->fromFile()->build();
+
+        // Construct a Google Ads client configured from a properties file and the
+        // OAuth2 credentials above.
+        $googleAdsClient = (new GoogleAdsClientBuilder())
+            ->fromFile()
+            ->withOAuth2Credential($oAuth2Credential)
+            ->build();
+
+        try {
+            self::runExample(
+                $googleAdsClient,
+                $options[ArgumentNames::CUSTOMER_ID] ?: self::CUSTOMER_ID,
+                $options[ArgumentNames::BILLING_SETUP_ID] ?: self::BILLING_SETUP_ID
+            );
+        } catch (GoogleAdsException $googleAdsException) {
+            printf(
+                "Request with ID '%s' has failed.%sGoogle Ads failure details:%s",
+                $googleAdsException->getRequestId(),
+                PHP_EOL,
+                PHP_EOL
+            );
+            foreach ($googleAdsException->getGoogleAdsFailure()->getErrors() as $error) {
+                /** @var GoogleAdsError $error */
+                printf(
+                    "\t%s: %s%s",
+                    $error->getErrorCode()->getErrorCode(),
+                    $error->getMessage(),
+                    PHP_EOL
+                );
+            }
+            exit(1);
+        } catch (ApiException $apiException) {
+            printf(
+                "ApiException was thrown with message '%s'.%s",
+                $apiException->getMessage(),
+                PHP_EOL
+            );
+            exit(1);
+        }
+    }
+
+    /**
+     * Runs the example.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param int $billingSetupId the billing setup ID to filter the invoices on
+     */
+    public static function runExample(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        int $billingSetupId
+    ) {
+        // Gets the date one month before now.
+        $lastMonth = strtotime('-1 month');
+
+        // [START GetInvoices]
+        // Issues the request.
+        $response = $googleAdsClient->getInvoiceServiceClient()->listInvoices(
+            $customerId,
+            ResourceNames::forBillingSetup($customerId, $billingSetupId),
+            // The year needs to be 2019 or later.
+            date('Y', $lastMonth),
+            MonthOfYear::value(strtoupper(date('F', $lastMonth)))
+        );
+        // [END GetInvoices]
```

Comment:
This is not exactly related to the example, but WDYT about having the explanation of this kind of region tags somewhere in our DevSite?
This would be a bit more friendly for any code example users.

Reply:
I like the idea. I could add a note on this [page](https://developers.google.com/google-ads/api/docs/samples) for reference, what do you think?

Reply:
SGTM.

Reply:
Done, let's continue this discussion offline.